### DISCUSSION TITLE
Rename failed - it also renamed the parent folders.

### DISCUSSION THREAD

**Discussion by jeffward01** [ModernRonin/ProjectRenamer#44] [General]
I have a directory structure of this:

* `targetDirectory/mySolution.sln`
* `targetDirectory/LibMain/lib/MyProject.RemoveThis.csproj`

When I perform the project rename command:
```bash
renamproject MyProject.RemoveThis MyProject
```

It completes successfully... However the issue is that it also renames the parent directory:

* `targetDirectory/LibMain/MyProject/MyProject.csproj`

The `lib` parent directory is changed to the name of the project, which in this case is `MyProject`

**Current:**

* Original Path: * `targetDirectory/LibMain/lib/MyProject.RemoveThis.csproj`
* After Rename Path: * `targetDirectory/LibMain/MyProject/MyProject.csproj`

**Expected:**

* Original Path: * `targetDirectory/LibMain/lib/MyProject.RemoveThis.csproj`
* After Rename Path: * `targetDirectory/LibMain/lib/MyProject.csproj`

Thanks

**Comment by ModernRonin** [comment]
this is per design - see the readme:

> This tool takes care of this for you, provided your use-case follows a set of fairly common practices:
[...]
your csproj files have the same name as the folder in which they reside together with accompanying source code

that being said, one could add an option that tells renamer to not expect said convention - I'm open for a PR :-)

**Reply by ModernRonin** [reply]
@jeffward01 I converted this to a discussion, so it's under discussions now rather than under issues
